addons/survey/views/survey_templates_print.xml

```diff
@@ -34,12 +34,52 @@
+                                            <t t-if="answer_lines and answer_lines.survey_id.scoring_type != 'no_scoring'">
+                                                <t t-set="answer_line" t-value="answer_lines[0]"/>
+                                                <t t-if="answer_line.skipped">
+                                                    <!-- question was skipped, display an orange block with the text "Skipped" in place of the answer -->
+                                                    <div class="row no-gutters o_survey_questions">
+                                                        <t t-set="answer_bg_class" t-value="'o_survey_question_skipped'"/>
+                                                        <div t-attf-class="col-12 col-md-6 col-lg-4 rounded pl-4 {{answer_bg_class}}">
+                                                            <input type="text"
+                                                                t-attf-class="form-control font-italic o_survey_question_{{question.question_type}} bg-transparent rounded-0 p-0"
+                                                                name="skipped_answers" value="Skipped"/>
+                                                        </div>
+                                                    </div>
+                                                </t>
+                                                <t t-else="">
+                                                    <t t-if="question.question_type in ['numerical_box', 'date', 'datetime']">
+                                                        <div class="row no-gutters o_survey_questions">
+                                                            <t t-set="answer_bg_class" t-if="answer_lines[0].answer_is_correct" t-value="'bg-success'"/>
+                                                            <t t-set="answer_bg_class" t-else="" t-value="'bg-danger'"/>
+                                                            <div t-attf-class="col-12 col-md-6 col-lg-4 rounded pl-4 {{answer_bg_class}}">
+                                                                <t t-call="survey.survey_scored_fields"/>
+                                                            </div>
+                                                        </div>
+                                                    </t>
+                                                    <t t-else="">
+                                                        <t t-call="survey.survey_text_fields"/>
+                                                    </t>
+                                                </t>
+                                                <t t-if="question.question_type in ['numerical_box', 'date', 'datetime'] and question['answer_%s' % question.question_type] and not answer_line.answer_is_correct">
```

Comment:
Yes, it will show the correct answer below the user's answer, that's what the spec is. Just like this!

![Screenshot from 2022-04-11 15-18-21](https://user-images.githubusercontent.com/72728567/162712951-371d7b3d-df40-4018-ad59-b2473cfbf47a.png)

Reply:
Never mind then, we're good on that one :smile: